### PR TITLE
Expanded setup script to cover Fedora as well.

### PR DESCRIPTION
**Pull request by matejak**
Allow setup of labs for Fedora hosts, so one can taste the labs without having to possess RHEL setup.

The RHEL8 path should be still preserved.

In order to test it, you need to have a Fedora VM - the best option is to have SSH keys set up so you can log in as a superuser, and that you have an entry in SSH config that maps the VM to `FEDORA_VM_HOSTNAME`.

Then, set your system up by running

```
ansible-playbook -i FEDORA_VM_HOSTNAME, system_setup.yml
ansible-playbook -i FEDORA_VM_HOSTNAME, labs_setup.yml
```

Pay attention at the comma after the hostname. If you connect as a passwordless sudoer, you may need to add the `-K` option to the playbook invocation and supply an arbitrary input as "sudo password".

### CODE REVIEW

**Comment by ggbecker** [comment]
One note, I had to use the ansible-playbook option as well to run the playbooks:
```
       -k, --ask-pass
          ask for connection password
```
because I didn't have any public key authentication setup on the VM.

I think it would be a good idea to have some sort of documentation on how to setup the VM and execute the playbooks. This way we can point people to this documentation when setting up the VM prior doing the labs (fedora only). What do you think?

**Comment by ggbecker** [comment]
We also need to update the location of tests scenarios directories in the lab setup, especially the lab5_oval setup.

File: 2019Labs/CustomSecurityContent/setup/system_setup.yml [hunk 1]
```diff
@@ -6,11 +6,12 @@
   tasks:
   - name: Enable Ansible repository
     yum_repository:
-      name: ansible-2.8-for-rhel-8-x86_64-rpms
+      name: ansible-2.9-for-rhel-8-x86_64-rpms
```
Comment (ggbecker) [inline 2019Labs/CustomSecurityContent/setup/system_setup.yml]: you changed here to 2.9 but the description and baseurl still uses 2.8. Is it correct?

Comment (matejak) [reply 2019Labs/CustomSecurityContent/setup/system_setup.yml]: Good catch, fixed!

**Pull request merged**